
Note: Command line contains non-control variables:
* DESTDIR=/export/home/gweiss/git/oi-userland/components/runtime/openjdk-21-21.0.10/build/prototype/i386
Make sure it is not mistyped, and that you intend to override this variable.
'make help' will list known control variables.

Note: Command line contains non-control variables:
* DESTDIR=/export/home/gweiss/git/oi-userland/components/runtime/openjdk-21-21.0.10/build/prototype/i386
Make sure it is not mistyped, and that you intend to override this variable.
'make help' will list known control variables.

--- jdk25u-jdk-25.0.3-ga/make/PreInitSupport.gmk.old
+++ jdk25u-jdk-25.0.3-ga/make/PreInitSupport.gmk
@@ -43,7 +43,7 @@
 
 # All known make control variables; these are handled in other makefiles
 MAKE_CONTROL_VARIABLES += JDK_FILTER SPEC_FILTER \
-    TEST TEST_JOBS JTREG GTEST MICRO TEST_OPTS TEST_VM_OPTS TEST_DEPS
+    TEST TEST_JOBS JTREG GTEST MICRO TEST_OPTS TEST_VM_OPTS TEST_DEPS DESTDIR
 
 ALL_CONTROL_VARIABLES := $(INIT_CONTROL_VARIABLES) $(MAKE_CONTROL_VARIABLES)
 
